src/main/resources/hudson/maven/Messages.properties

```diff
@@ -55,7 +55,7 @@ MavenProbeAction.DisplayName=Monitor Maven Process
 MavenProcessFactory.ClassWorldsNotFound=No classworlds*.jar found in {0} -- Is this a valid maven directory?
 
 MavenRedeployer.DisplayName=Deploy to Maven repository
-MavenVersionCallable.MavenHomeDoesntExist=Maven Home {0} doesn\u2019t exist
+MavenVersionCallable.MavenHomeDoesntExist=Maven Home {0} doesn't exist
```

Comment:
I was just going on the build error message (which I'm afraid I don't recall) followed by various web search results. The resulting web pages variously seemed be saying that \u2019 is an incorrect/obsolete/ambiguous code point for the right apostrophe / right single quote character and there was some question as to the languages in which its use is appropriate. Anyway, it was the only way I could find to make the build run on Windows, plus this properties file now consistently uses standard ASCII single quote characters (\u0027) throughout.